web/views/sleep_tracker_view.ex

```diff
@@ -22,10 +22,35 @@ defmodule Healthlocker.SleepTrackerView do
     if Kernel.length(past_week) == 0 do
       0
     else
-      total_slept / Kernel.length(past_week)
+      format_average_sleep(total_slept / Kernel.length(past_week))
     end
   end
 
+  defp hours_mins(humanized_str) do
+    humanized_str
+      |> String.replace(" hours,", "h")
+      |> String.replace(" minutes", "m")
+      |> String.split(",")
+      |> List.first
+  end
+
+  @doc """
+  format_average_sleep
+
+  ## Example
+
+      iex> import Healthlocker.SleepTrackerView
+      iex> format_average_sleep(7.375)
+      "7h 22m"
+
+  """
+
+  def format_average_sleep(num) do
+    Duration.from_hours(num)
+      |> Timex.Format.Duration.Formatter.format(:humanized)
+      |> hours_mins()
```

Comment:
Ahh that sucks, yeah looking at it the `Duration` formatters don't look as flexible 😞  We could maybe write our own, like [`Duration.Formatters.Humanized`](https://github.com/bitwalker/timex/blob/master/lib/format/duration/formatters/humanized.ex), but I don't think it'll be worth it.